subworkflows/local/utils_nfcore_phaseimpute_pipeline/main.nf

```diff
@@ -448,25 +486,21 @@ def validateInputParameters() {
         assert params.input : "No input provided"
     }
 
-    // Check that posfile and chunks are provided when running impute only. Steps with panelprep generate those files.
+    // Check that posfile and panel are provided when running impute only
```

Comment:
Maybe it would be useful to assert number of panels provided?

```
def panel_ids = panel_id.toList()
assert panel_ids.size() <= 1 : "Multiple panel IDs detected (${panel_ids.join(',')}) but panel/chunks/posfile were not explicitly provided"
```